core/src/main/scala/kafka/tools/SimplePropertiesParser.scala

```diff
@@ -0,0 +1,25 @@
+package kafka.tools
+
+import scala.collection.JavaConversions._
+import scala.collection.immutable
+import scala.util.control.NonFatal
+
+object SimplePropertiesParser {
+
+  /**
+   * Take a simple string of the form key=val,key=val,key=val and parse into a map. Useful for command line
+   * arguments.
+   * @param freeform String of format "key=val,key=val,key=val"
+   * @return
+   */
+  def propsFromFreeform(freeform: String) : java.util.Map[String,String] = {
+    try {
+      val argProps: immutable.Map[String, String] = freeform.split(",")
+        .map(_.split("="))
+        .map({ case Array(k, v) => (k, v)}).toMap
```

Comment:
```.map { case Array(k, v) => (k, v) }.toMap``` is fine, you don't need both `(` and `{`. I think you should have two `case` statement, the second one being a `case _`. Then you don't need the `catch` right? Also, instead of `JavaConversions`, prefer `JavaConverters` which means you can use the `asJava` or `asScala` methods directly.